client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -3,8 +3,14 @@
 
 'use strict';
 
-MetadataCtrl.$inject = ['$scope', 'desks', 'metadata', '$filter', 'privileges', 'datetimeHelper'];
-function MetadataCtrl($scope, desks, metadata, $filter, privileges, datetimeHelper) {
+MetadataCtrl.$inject = [
+    '$scope', 'desks', 'metadata', '$filter', 'privileges', 'datetimeHelper',
+    'preferencesService'
+];
+function MetadataCtrl(
+    $scope, desks, metadata, $filter, privileges, datetimeHelper,
+    preferencesService
+) {
```

Comment:
I followed the pattern I sometimes see in `if` statements with longer conditionals, for example:
```js
if (foo.property.something || bar.baz.another.something ||
    (sky.is.blue && sun.is.shining) ||
    button.disabled || items.length > 0
) {
    // do something
    // and some more
    // ...
}
```

The goal was to keep the opening curly bracket indented to the left so that the "block head" is easily visually distinguished from the block body. It doesn't look ideal though, thus alternative suggestions welcome.

P.S.: Would it look better if the dependencies were spread more evenly across the lines, e.g. like this?
```js
function MetadataCtrl(
    $scope, desks, metadata, $filter, privileges,
    datetimeHelper, preferencesService
) {
    // ...
};
```